packages/catalog-search/src/tests/SearchSuggestions.test.jsx

```diff
@@ -67,7 +67,6 @@ describe('<SeachSuggestions />', () => {
       autoCompleteHits={fakeSuggestionsDataEmptyAuthoringOrgs.hits}
       handleSubmit={handleSubmit}
     />);
-    expect(screen.getByText('Courses')).not.toBeNull();
```

Comment:
made the decision that we shouldn't have an empty half container with title and no results if no results are present